commands/lyrics.js

```diff
@@ -0,0 +1,44 @@
+const { MessageCollector } = require('discord.js');
+const helpEmbed = require('../embeds/helpEmbed.js');
+const { HELP, HELP_SHORT } = require('../assets/flags.json');
+const Utils = require('../utils/utils.js');
+const { getCCLyricsUrl } = require('../utils/cclyrics.js');
+
+const configuration = {
+    enabled: true,
+    name: 'lyrics',
+    aliases: [ 'ccl' ],
+    description: `Send a link to a song's lyrics based on the search term.`,
+    usage: 'lyrics <SEARCH_TERM>',
+};
+
+module.exports = {
+    conf: configuration,
+
+    run: async (client, message, args) => {
+        const searchTerm = args.join(' ');
+
+        if (args[0] === HELP || args[0] === HELP_SHORT || args.length < 1) {
+            helpEmbed(message, configuration);
+            Utils.errAndMsg(message.channel, 'Invalid arguments.');
```

Comment:
I think we should either:
- just remove the error altogether for simplicity? If the user doesn't give enough args, it's probably enough to just give them a help embed. I think most command line tools does that.
- move the error to `else if (args.length < 1)`

I feel switch isn't designed for this specific situation 😅

Reply:
I like option 1, since I think if you get the help window, I think it should imply that you did something wrong? Should we make a silent errAndMsg then?

Reply:
Im for option 2 lol

Reply:
you don't sacrifice functionality for code cleanliness

Reply:
I'd agree if the functionality is useful. In this case it's standard behaviour for commandline to just output the help prompt for incorrect args. I think we shouldn't accrue tech debt to add fluff.